lib/jose-utils.js

```diff
@@ -367,5 +367,15 @@ Utils.sha256 = function(str) {
 Utils.isCryptoKey = function(rsa_key) {
   // Some browsers don't expose the CryptoKey as an object, so we need to check
   // the constructor's name.
-  return rsa_key.constructor.name == 'CryptoKey';
+  if (rsa_key.constructor.name == 'CryptoKey') {
```

Comment:
By "is defined", do you mean `typeof CryptoKey !== 'undefined'`? I am not really strong at all on this entire JavaScript ecosystem.